plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/DeployPropertyPage.java

```diff
@@ -111,8 +111,10 @@ private DeployPreferencesPanel getPreferencesPanel(Composite container) throws C
     IFacetedProject facetedProject = ProjectFacetsManager.create(project);
 
     if (AppEngineStandardFacet.hasAppEngineFacet(facetedProject)) {
+      setTitle(Messages.getString("standard.page.title"));
```

Comment:
realizing I don't like the name of this method. Should probably be just hasFacet; but that's for another PR